### DISCUSSION TITLE
Can Intel iGPU (QSV) be leveraged in the UnRAID Docker Container

### DISCUSSION THREAD

**Discussion by chutson92** [stashapp/stash#5306] [Q&A] [answered]
Can Intel iGPU (QSV) be leveraged in the UnRAID Docker Container

*### Scope*
I am wondering how/if an Intel iGPU (QSV) can be leveraged in the UnRAID Docker Container?  Not only would this be better for transcoding to a client, but it should also speed up marker generation, previews, etc.  Perhaps I missed something in the documentation to enable this, but I'm not able to find anything.

**Comment by feederbox826** [comment]
https://feederbox.cc/hwaccel

If you're on ARC, then you'll need to install the drivers on your host

**Reply by feederbox826** [reply, marked as answer]
You can also follow the instructions here: https://github.com/stashapp/stash/issues/5726
And follow this issue for hardware accelerated preview generation
